portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,43 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sabrina Snider | Software Engineer</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>Sabrina's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <img src="images/tux.jpg" height="600">
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="navbar" class="sticky">
+        <div id="nav-header">
+          <h1 id="nav-name">Sabrina Snider</h1>
+        </div>
+        <div id="nav-links">
+          <button class="nav-item">About Me</button>
+          <button class="nav-item">Work</button>
+          <button class="nav-item">Projects</button>
+        </div>
+      </div>
+
+      <div id="intro">
+        <h1 class="big-header-1">Hello, I'm Sabrina Snider.</h1>
+        <h2 class="sub-header-1">I like writing code.</h2>
+      </div>
+
+      <div id="smaller">
```

Comment:
When naming ids/classnames, try to describe what the section is doing, not a "presentational attribute" like smaller text, etc. This will reduce refactoring if you decide to change styles later.

Here's what the style guide says

> Instead of presentational or cryptic names, always use ID and class names that reflect the purpose of the element in question, or that are otherwise generic.
> 
> Names that are specific and reflect the purpose of the element should be preferred as these are most understandable and the least likely to change.
> 
> Generic names are simply a fallback for elements that have no particular or no meaning different from their siblings. They are typically needed as “helpers.”

Reply:
Good point! I ended up removing the div entirely and I found I could accomplish the same goal with margins.